src/components/player/cue-bar.tsx

```diff
@@ -82,7 +82,17 @@ const MutePopupButton: React.FC<any> = () => {
       className="text-gray-400 rounded flex-nowrap flex items-center text-xs"
       onClick={() => setPlayerPrefs({muteNotes: !muteNotes})}
     >
-      {muteNotes ? <IconVolumeOn /> : <IconVolumeOff />}
+      {muteNotes ? (
+        <>
+          <span className="pr-1">unmute notes</span>
```

Comment:
I switched the icons to show current state and added text prompt for what clicking will do. I think these need aria labels and a11y consideration?